ios/ble/Utility/TransferHandler.swift

```diff
@@ -99,8 +101,10 @@ class TransferHandler {
 
     private func sendResponseSize(size: Int) {
         let decimalString = String(size)
-        let d = decimalString.data(using: .utf8)
-        Central.shared.write(serviceUuid: Peripheral.SERVICE_UUID, charUUID: NetworkCharNums.RESPONSE_SIZE_CHAR_UUID, data: d!)
+        if let data = decimalString.data(using: .utf8) {
+            print(data)
```

Comment:
change print statements to os_log statements

Reply:
this log is for developer purpose, I'll remove this.